frontend/public/components/build-pipeline.tsx

```diff
@@ -40,9 +41,11 @@ const BuildSummaryStatusIcon: React.SFC<BuildSummaryStatusIconProps> = ({ status
 
 const BuildLogLink: React.SFC<BuildLogLinkProps> = ({ obj }) => {
   const link = getJenkinsLogURL(obj);
-  return link && <div className="build-pipeline__link">
-    <a href={link} className="build-pipeline__log-link" target="_blank" rel="noopener noreferrer">View Log</a>
-  </div>;
+  return link
+    ? <div className="build-pipeline__link">
+      <a href={link} className="build-pipeline__log-link" target="_blank" rel="noopener noreferrer">View Log</a>
+    </div>
+    : null;
```

Comment:
nit: this could be `return link && <div>...</div>`

Reply:
That is actually what was causing the bug in the first place. If we use that syntax, the render function does not return anything if there is no link, causing React to throw an exception.

Reply:
Oops, d'oh. Of course. Please disregard :)